carma-messenger-core/j2735_convertor/README.md

```diff
@@ -0,0 +1,3 @@
+# j2735_convertor
+
+TODO for USER: Add description of package and link to confluence documentation.
```

Comment:
Add a quick description of this node and check if we have a confluence doc you can link. If there's no doc that's fine